data/json/items/tools.json

```diff
@@ -7438,7 +7438,7 @@
   
   {
     "id": "sauerkraut_ferment",
-    "type": "TOOL",
+    "type": "GENERIC",
```

Comment:
Please also remove the TOOL specific members "max_charges", "initial_charges", "charges_per_use", "turns_per_charge", "ammo" and "revert_to". They have no meaning in a GENERIC item type.

There are some items in the More_Survival_Tools and blazemod mod that have the same problem.